<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tof_powder_plot</class>
 <widget class="QWidget" name="tof_powder_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1965</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="l_min">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_max">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sl_min">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sl_max">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
       <property name="sliderPosition">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="plot_layout">
     <item>
      <layout class="QHBoxLayout" name="plot_head_layout">
       <item>
        <widget class="QComboBox" name="combB_colormap">
         <property name="iconSize">
          <size>
           <width>400</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cB_log_scale">
         <property name="text">
          <string>logScale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cB_reverse_colormap">
         <property name="text">
          <string>reverse map</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
